src/main/java/com/laborhelper/enterprise/LaborHelperController.java

```diff
@@ -8,11 +8,11 @@ public class LaborHelperController {
 
     /**
      * Handle the root (/) endpoint and return a start page
-     * @return
+     * @return This method returns an HTML page
      */
     @RequestMapping("/")
     public String index() {
-        return "start";
+        return "index";
```

Comment:
The default Spring Initializr has the Thymeleaf template name as start, when it really should be your index page for naming sake